pkg/printers/internalversion/printers.go

```diff
@@ -376,7 +376,6 @@ func printPod(pod *api.Pod, options printers.PrintOptions) ([]metav1alpha1.Table
 			reason = fmt.Sprintf("Init:%d/%d", i, len(pod.Spec.InitContainers))
 			initializing = true
 		}
-		break
```

Comment:
It's in a for loop, this is how initializers work.  We stop as soon as we see an initializer.